boomonkeyboo/scratch_project
# Chat components + Integrated with backend

Issue Type
☐ Bug
☑︎ Feature
☐ Tech Debt

Description

Structure the Chat components + Integrated with backend

Steps to Validate Feature

Click around on text fields and inspect the results in the console

Previous behavior
N/A

Expected behavior
Console.logging some test messages
